packages/EasyBankFiles/src/Parsers/DirectEntry/Parser.php

```diff
@@ -5,67 +5,79 @@
 namespace EonX\EasyBankFiles\Parsers\DirectEntry;
 
 use EonX\EasyBankFiles\Parsers\AbstractLineByLineParser;
-use EonX\EasyBankFiles\Parsers\DirectEntry\Results\Header;
-use EonX\EasyBankFiles\Parsers\DirectEntry\Results\Trailer;
-use EonX\EasyBankFiles\Parsers\DirectEntry\Results\Transaction;
+use EonX\EasyBankFiles\Parsers\DirectEntry\Results\Batch;
+use EonX\EasyBankFiles\Parsers\DirectEntry\Results\Batch\Header;
+use EonX\EasyBankFiles\Parsers\DirectEntry\Results\Batch\Trailer;
+use EonX\EasyBankFiles\Parsers\DirectEntry\Results\Batch\TransactionTypePayment;
+use EonX\EasyBankFiles\Parsers\DirectEntry\Results\Batch\TransactionTypeRefusal;
+use EonX\EasyBankFiles\Parsers\DirectEntry\Results\Batch\TransactionTypeReturn;
 use EonX\EasyBankFiles\Parsers\Error;
 
+/**
+ * @SuppressWarnings(PHPMD.ExcessiveClassComplexity) We can't reduce the overall complexity
+ */
 final class Parser extends AbstractLineByLineParser
```

Comment:
Could you please outline the difference between the "old" parser and the new one usage in the PR description? So we can see an example of the batch feature of the new parser.